src/main/java/org/ibp/api/domain/common/LabelPrintingStaticField.java

```diff
@@ -62,4 +66,24 @@ public void setFieldId(final Integer fieldId) {
 		return availableStaticFieldIds;
 	}
 
+	public static LabelPrintingStaticField getByFieldId(final Integer id) {
+		if (id == null) {
+			return NONEXISTENT;
+		}
+
+		if (!LabelPrintingStaticField.STATIC_FIELD_MAP.containsKey(id)) {
```

Comment:
You can simplify this by:

```
private static final Map<Integer, LabelPrintingStaticField> STATIC_FIELD_MAP_LOOKUP = new HashMap<>();
static {
		for (final LabelPrintingStaticField sf : EnumSet.allOf(LabelPrintingStaticField.class)) {
			STATIC_FIELD_MAP_LOOKUP.put(sf.getFieldId(), sf);
		}
	}
```
```
public static Optional<LabelPrintingStaticField> getByFieldId(final Integer id) {
		if (id == null) {
			return Optional.empty();
		}
		if (LabelPrintingStaticField.STATIC_FIELD_MAP_LOOKUP.containsKey(id)) {
			return Optional.of(STATIC_FIELD_MAP_LOOKUP.get(id));
		}
		return Optional.empty();

	}
```
Use optional instead of creating enum for non existent field.